### PR TITLE
Add mariadb coordinator

### PR DESCRIPTION
**Pull request by spectro30**
Signed-off-by: Md. Alif Biswas <alif@appscode.com>

### CODE REVIEW

File: pkg/controller/statefulset.go [hunk 1]
```diff
@@ -552,6 +639,54 @@ func upsertSharedScriptsVolume(statefulSet *apps.StatefulSet) *apps.StatefulSet
...
+				MountPath: "/run-script",
+			}
+			volumeMounts := container.VolumeMounts
+			volumeMounts = core_util.UpsertVolumeMount(volumeMounts, configVolumeMount)
+			statefulSet.Spec.Template.Spec.Containers[i].VolumeMounts = volumeMounts
+		}
+
+	}
+	for i, initContainer := range statefulSet.Spec.Template.Spec.InitContainers {
+		if initContainer.Name == "mariadb-init" {
```
Comment (tamalsaha) [inline pkg/controller/statefulset.go:665]: `MariaDBInitContainerName`

Comment (spectro30) [reply pkg/controller/statefulset.go:665]: fixed

File: pkg/controller/statefulset.go [hunk 2]
```diff
@@ -61,6 +61,7 @@ func (c *Reconciler) ensureStatefulSet(db *api.MariaDB) (kutil.VerbType, error)
 			in.Spec.Selector = &metav1.LabelSelector{
 				MatchLabels: db.OffshootSelectors(),
 			}
+			in.Spec.PodManagementPolicy = "Parallel"
```
Comment (faem) [inline pkg/controller/statefulset.go:64]: Use the constant `ParallelPodManagement`

Comment (spectro30) [reply pkg/controller/statefulset.go:64]: added

**Review by faem — APPROVED** [review]

**Pull request merged**